src/lib/trivy/trivy-collector.spec.ts

```diff
@@ -59,6 +60,19 @@ describe('TrivyCollector', () => {
       expect(collector.parseResults).toHaveBeenCalledTimes(1);
       expect(collector.parseResults).toHaveBeenCalledWith('TEST_OUTPUT');
     });
+
+    it('should error when image cannot be found', async () => {
+      collector._argv = {
+        'image-name': 'TEST_IMAGE',
+      } as any;
+
+      theSpy.and.returnValue(new Promise((resolve, reject) => {
```

Comment:
Rather than referencing it this way, we should either just reference the `collector.spawn` property directly or reassign it `collector.spawn = jasmine.createSpy()...`